config/i18n-tasks.yml

```diff
@@ -0,0 +1,129 @@
+# i18n-tasks finds and manages missing and unused translations: https://github.com/glebm/i18n-tasks
```

Comment:
I think we can remove all that commented boilerplate code to improve the readability

Reply:
No problem. I shall tidy it up.